src/Connectors/src/ConnectorBase/DocumentDB/MongoDb/MongoDbHealthContributor.cs

```diff
@@ -25,18 +26,29 @@ public static IHealthContributor GetMongoDbHealthContributor(IConfiguration conf
             var info = configuration.GetSingletonServiceInfo<MongoDbServiceInfo>();
             var mongoOptions = new MongoDbConnectorOptions(configuration);
             var factory = new MongoDbConnectorFactory(info, mongoOptions, MongoDbTypeLocator.MongoClient);
-            return new MongoDbHealthContributor(factory, logger);
+            var timeout = 5000;
```

Comment:
Should default `timeout` be externally configurable?  Or at least a named constant?

Reply:
It could be a named constant, but I think making the *default* be externally configurable wouldn't be worth the effort... this was at least an improvement over having it inline and non-configurable